File: CarRental/src/main/kotlin/mas/models/Company.kt
package mas.models

import java.util.*
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id

@Entity(name = "company")
data class Company(

    @Id
    val id: UUID = UUID.randomUUID(),

    val name: String,

    @Column(name = "phone_number")
    val phoneNumber: String,

    val description: String,

    val image: String

)

File: mas-ui/src/models/Company.ts
export{Company}

class Company {

    id: string
    name: string
    image: string
    phoneNumber: string
    description: string
    relations: number

    constructor(id: string, name: string, image: string, phoneNumber: string, description: string, relations: number) {
        this.id = id
        this.name = name
        this.image = image
        this.phoneNumber = phoneNumber
        this.description = description
        this.relations = relations
    }

}
